torch/_inductor/fx_passes/serialized_patterns/addmm_pattern.py

```diff
@@ -0,0 +1,53 @@
+# mypy: ignore-errors
+
+# noqa: F401, E501
+# This is an auto-generated file. Please do not modify it by hand.
+# To re-generate, run:
+# cd ~/pytorch && python
+# torchgen/fuse_attention_patterns/gen_attention_patterns.py
```

Comment:
Not relevant here right ?

Reply:
Which part? It's auto-generated and you can use that script to regenerate it (the script is poorly named now since it regenerates ALL patterns, not just fuse_attention_patterns - maybe that's your issue?)

Reply:
Ah, I mistakenly thought it was only generating the attention patterns, and not yet the mm patterns. Maybe we can rename the file ?